src/main/java/uk/gov/pay/connector/wallets/model/WalletPaymentInfo.java

```diff
@@ -32,6 +32,13 @@ public WalletPaymentInfo(String lastDigitsCardNumber, String brand, PayersCardTy
         this.email = email;
     }
 
+    public WalletPaymentInfo(String lastDigitsCardNumber, String brand, PayersCardType cardType, String cardholderName, String email, String acceptHeader, String userAgentHeader, String ipAddress) {
```

Comment:
Can we move parameters from one line into multilines. We don't have to scroll to the right hand side to see all the parameters.
```suggestion
    public WalletPaymentInfo(String lastDigitsCardNumber,
                             String brand,
                             PayersCardType cardType,
                             String cardholderName,
                             String email,
                             String acceptHeader,
                             String userAgentHeader,
                             String ipAddress) {
```

Reply:
Done